src/views/virtualmachines/actions/components/VirtualMachineMigration/constants.ts

```diff
@@ -0,0 +1,84 @@
+import {
+  K8sModel,
+  K8sResourceCommon,
+  K8sResourceCondition,
+} from '@openshift-console/dynamic-plugin-sdk';
+
+export type MigPlan = K8sResourceCommon & { spec: any; status?: { suffix: string } };
+
+export const MigPlanModel: K8sModel = {
```

Comment:
why mig?

Reply:
this is the resource name MigPlan from MTC